solutions/usaco-1088.mdx

```diff
@@ -11,3 +11,151 @@ author: Melody Yu (Video), Tyler Xiao (Video)
 
 Note: The video solution might not be the same as other solutions. Code in Java.
 <Youtube id="7eMHxpDQJEw" />
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2)$
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <algorithm>
+using namespace std;
+
+int main()
+{
+	int n;
+	cin >> n;
+	int grid[1000][1000];
```

Comment:
use vectors please

Reply:
why? i think this is fine...

Reply:
think local arrays can do some weird crap with memory
if you're gonna keep this at least have a `MAX_N` constant or something

Reply:
this is fine... if concerned about memory then move it to the global scope